composer.json

```diff
@@ -17,6 +17,7 @@
         "doctrine/persistence": "1.3.2",
         "knplabs/knp-time-bundle": "1.9.0",
         "lexik/maintenance-bundle": "2.1.4",
+        "lcobucci/jwt": "4.2.0 || 4.2.1 || 4.3.0",
```

Comment:
```suggestion
        "lcobucci/jwt": ">=4.2.0",
```

Reply:
I wasn't sure about ranges with no upper limit 😅

Reply:
Adjusted in [cefbfb4](https://github.com/contao/conflicts/pull/43/commits/cefbfb4f32081b11872770519fc6f87e6914d51d)

Reply:
Usually we try to avoid ranges with no upper limit, however, since there are no plans to revert the changes on their side, all future versions are broken for us until we fix the issue on our side (and then remove the conflict again).